src/engraving/rendering/dev/measurelayout.cpp

```diff
@@ -796,6 +796,9 @@ void MeasureLayout::createMultiMeasureRestsIfNeed(MeasureBase* currentMB, Layout
             }
             firstMeasure->setMMRestCount(0);
             ctx.mutState().setMeasureNo(mno);
+            if (lastMeasure->endTick() > ctx.state().endTick()) {
+                ctx.mutState().setEndTick(lastMeasure->endTick());
```

Comment:
if a multi-measure rest is removed, we need ensure the layout tick range covers all the underlying measures